packages/slice-machine/test/core/libraries/libraries.test.ts

```diff
@@ -133,7 +134,9 @@ test("it finds non local libs", () => {
 
   const result = libraries(TMP, [libName]);
   expect(result[0].isLocal).toEqual(false);
-  expect(result[0].components[0].pathToSlice).toEqual(`${libName}/slices`);
+  expect(result[0].components[0].pathToSlice).toEqual(
+    path.join(`${libName}/slices`)
```

Comment:
shouldn't this fail on windows because of the `/`?

Reply:
POSIX path delimiters (`/`) are interpreted correctly by Windows (`win32` platforms) in the case of path resolution here and Node imports.
In any case, this test is not relevant anymore as it's now the responsibility of plugins to perform such resolutions, and likely to be removed in #812